test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml

```diff
@@ -15,6 +15,16 @@ spec:
     spec:
       serviceAccountName: csi-gce-pd-controller-sa
       containers:
+        - name: csi-snapshotter
+          # TODO: replace with official 2.0.0 release when ready
+          image: quay.io/k8scsi/csi-snapshotter:v2.0.0-rc2
+          args:
+            - "--v=5"
+            - "--csi-address=/csi/csi.sock"
+          imagePullPolicy: Always
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
         - name: csi-provisioner
           image: gcr.io/gke-release/csi-provisioner:v1.4.0-gke.0
```

Comment:
I believe provisioner also has to be updated too? cc @xing-yang

Reply:
Yes.  Use quay.io/k8scsi/csi-provisioner:v1.5.0-rc1

Reply:
updated. thx